### PR TITLE
Flexbox in Solid

### PR DESCRIPTION
**Pull request by emilybrick**
cut a new branch and pulled the necessary changes over from the old one!

### CODE REVIEW

File: docs/flexbox.html [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+    <svg class="page-message__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 38">
+      <path d="M17.5 8.7h3v12.6h-3V8.7zM19 29.5c1.38 0 2.5-1.12 2.5-2.5s-1.12-2.5-2.5-2.5-2.5 1.12-2.5 2.5 1.12 2.5 2.5 2.5z"/>
+    </svg>
+    Flexbox classes are currently in beta, but we'd love to hear your thoughts. Send feedback to <a href="#">solid@buzzfeed.com</a>.</div>
+  </div>
+
+  <h2 class="xs-mb2 bold">Introducing Flexbox in Solid</h2>
+  <p>Flexbox is a way to efficiently lay out, align, and distribute items on a page. If you want to learn more about Flexbox and how to use it, there is plenty of good documentation around the web, including this <a href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">CSS Tricks article</a> and this <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Using_CSS_flexible_boxes">MDN spec</a>. One thing to note is that this system uses a flexible box layout model, which doesn't play particularly nice with our traditional grid model. When using Flexbox with Solid, be considerate about using it independently of other Solid layout classes, such as floats or block (as flex will overrride these).
```
Comment (allisonchefec) [inline docs/flexbox.html]: typo in "override"

**Comment by dreki** [comment]
We should add classes for the `flex:` CSS property. 
> The flex CSS property is a shorthand property specifying the ability of a flex item to alter its dimensions to fill available space. Flex items can be stretched to use available space proportional to their flex grow factor or their flex shrink factor to prevent overflow.

**Comment by dreki** [comment]
`align-self:` allows individual children override parent alignment. Could be useful to include.

**Comment by dreki** [comment]
Since we have `justify-content:` we may also want to provide `align-content:`. It does a similar job, but instead of justifying and spreading out empty space, it aligns children and puts the empty space to the opposite side, in one "chunk."

File: docs/_includes/nav.html [hunk 2]
```diff
@@ -36,8 +36,6 @@ <h1 class="xs-relative xs-py3 sm-py0 sm-mb4 solid-header">
             <li class="xs-py1"><a href="#colors" class="xs-block xs-px3 solid-nav__subitem">Colors</a></li>
             <li class="xs-py1"><a href="#layout" class="xs-block xs-px3 solid-nav__subitem">Layout</a></li>
             <li class="xs-py1"><a href="#borders" class="xs-block xs-px3 solid-nav__subitem">Borders</a></li>
-            <li class="xs-py1"><a href="#disabled-opacity" class="xs-block xs-px3 solid-nav__subitem">Disabled Opacity</a></li>
```
Comment (emilybrick) [inline docs/_includes/nav.html]: do we need this?

Comment (allisonchefec) [inline docs/_includes/nav.html]: i think we should leave it. it doesn't make sense in any of the other categories. i don't know why it was removed but i can add it back

**Pull request merged**